coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 88.48
+    "covered_percent": 34.94
```

Comment:
Didn't drop this much, did it?

Reply:
I'm not sure. Something weird with my testing environment, I think.

Reply:
So I deleted my coverage directory. Then ran rspec. It didn't generate a new file. Idk.

Reply:
Yes. Do you have a different suggestion? :(

Reply:
At some point we should hold an intervention for @mboneil10 's `rspec` installation. This has been a recurring theme across a few projects, if I'm not mistaken.